christfellowshipchurch/web-app-v2
# Feature feed events

Sets up events to the feature feed pattern. 

TO TEST:

Go to the `/events` page and you should see all the same content as is on the current live site
![image](https://user-images.githubusercontent.com/2528817/109874685-0ee60a80-7c35-11eb-8a21-f25739b8bd63.png)
